RecoLocalTracker/SiPixelRecHits/plugins/PixelCPEClusterRepairESProducer.cc

```diff
@@ -0,0 +1,80 @@
+#include "RecoLocalTracker/SiPixelRecHits/interface/PixelCPEClusterRepairESProducer.h"
+#include "RecoLocalTracker/SiPixelRecHits/interface/PixelCPEClusterRepair.h"
+#include "MagneticField/Engine/interface/MagneticField.h"
+#include "MagneticField/Records/interface/IdealMagneticFieldRecord.h"
+#include "Geometry/TrackerGeometryBuilder/interface/TrackerGeometry.h"
+#include "Geometry/Records/interface/TrackerDigiGeometryRecord.h"
+#include "Geometry/Records/interface/TrackerTopologyRcd.h"
+#include "DataFormats/TrackerCommon/interface/TrackerTopology.h"
+#include "CalibTracker/Records/interface/SiPixelTemplateDBObjectESProducerRcd.h"
+#include "CalibTracker/Records/interface/SiPixel2DTemplateDBObjectESProducerRcd.h"
+
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/ModuleFactory.h"
+#include "FWCore/Framework/interface/ESProducer.h"
+
+
+
+#include <string>
+#include <memory>
+
+using namespace edm;
+
+PixelCPEClusterRepairESProducer::PixelCPEClusterRepairESProducer(const edm::ParameterSet & p) 
+{
+  std::string myname = p.getParameter<std::string>("ComponentName");
+
+  //DoLorentz_ = p.getParameter<bool>("DoLorentz"); // True when LA from alignment is used
+  DoLorentz_ = p.existsAs<bool>("DoLorentz")?p.getParameter<bool>("DoLorentz"):false;
+
+  pset_ = p;
+  setWhatProduced(this,myname);
+
+  //std::cout<<" from ES Producer Templates "<<myname<<" "<<DoLorentz_<<std::endl;  //dk
```

Comment:
Please remove commented out lines (or replace with a LogDebug)

Reply:
OK, if you require this change now.